vpr/src/place/move_utils.h

```diff
@@ -36,6 +36,23 @@ enum class e_create_move {
     ABORT, //Unable to perform move
 };
 
+struct t_search_range {
```

Comment:
Comment this.

Reply:
Is this the same as a t_bb?

Reply:
Could update t_bb to have a constructor etc. if you find it's the same but less full featured.

Reply:
You are right. It is the same as t_bb. I will replace it with t_bb.